README.md

```diff
@@ -80,9 +81,9 @@ Click the link corresponding to your OS.
 ### 3. Launch the Plugin
 
 Once the napari window opens, go to **Plugins**.
-- If **napari-allencell-annotator: Annotator** is listed click it to launch. 
+- If **Annotator (napari-napari_allencell_annotator)** is listed click it to launch. 
```

Comment:
its just napari-allencell-annotator

Reply:
change the document in https://github.com/aics-int/napari-allencell-annotator/pull/105/commits/5f73f5e983dffcb583c2de0b4cc5cc10afe366e3